ci/tests/random_mixed_alloc.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+#include "gc.h"
+#include <cheriintrin.h>
+
+int main(void)
+{
+    int i,j;
+
+    srand(42);
+    GC_INIT();
+    for (i = 0; i < 10000000; ++i) {
+        unsigned int random_size = rand() % 1024;
+        size_t sz = random_size * sizeof(int *);
+        printf("[%s:%d]| [%u]  gc_malloc(%d)\n", __FUNCTION__, __LINE__,i ,  sz); fflush(NULL);
+        int **p = (int **) GC_MALLOC(sz);
+        printf("[%s:%d]| [%u]  gc_malloc(%d) -> %lp\n", __FUNCTION__, __LINE__,i ,  sz, p); fflush(NULL);
+        assert(*p == 0);
+        if (i % 100000 == 0)
```

Comment:
I guess that the `printf`s above are to help debugging, but I wasn't sure why this `if` is very helpful. Suggestions welcome!

Reply:
You are right. The `printfs` were for debugging. The if condition is only to give occasional debug output to ensure it is still running. I have removed the printfs and left in `if()` condition for the occasional status message in [2cc9b31d](https://github.com/capablevms/bdwgc/pull/11/commits/2cc9b31d999e78ceced82eadae95bcd57306d72a).